node/actors/bft/src/testonly/ut_harness.rs

```diff
@@ -224,15 +224,24 @@ impl UTHarness {
         ctx: &ctx::Ctx,
         msg: ReplicaPrepare,
     ) -> Signed<LeaderPrepare> {
+        let expected_validator_weight =
```

Comment:
use the weights from genesis().validators, don't recompute it

Reply:
Mhh, we can do that, but then a bug in weights distribution won't be catch in the asserts inside the for loop. Is that ok?

Reply:
that's ok